### ISSUE TITLE
Bug in  use-response-body (#a06576)

### ISSUE THREAD

**Issue by akshetpandey** [swipely/docker-api#276]
Not all Excon exception messages have .response method.

This causes the following exception:

```ruby
undefined method `response' for #<Excon::Errors::Timeout: read timeout reached>
```

Bug was introduced in: a06576566ba5bcc0a30104077f2ed1f312ec784d

**Comment by tlunter** [comment]
Would make sense that a timeout doesn't have a response.  Will get a patch version out for that immediately.

**Comment by tlunter** [comment]
Fixed in v1.21.2, the other errors should all have a response attached.
